tmp/cmake/visitConfig.cmake

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
This is in a `tmp` dir. Was this intended to be part  of this PR?

Reply:
There is a comment in the Description regarding this tmp dir.